etc/zinit/vmd.yaml

```diff
@@ -0,0 +1,3 @@
+exec: vmd -broker unix:///var/run/redis.sock
+after:
+  - node-ready
```

Comment:
shouldn't it also depend on `networkd` and `storaged` ?

Reply:
yes, this should be set to
```yaml
- boot
- networkd
```